var/spack/repos/builtin/packages/beast1/package.py

```diff
@@ -26,6 +26,7 @@ def url_for_version(self, ver):
 
     def setup_environment(self, spack_env, run_env):
```

Comment:
This needs to be updated to the new API to get the tests to pass:
```suggestion
    def setup_run_environment(self, env):
```

Reply:
Is there a quick way that I can check for things like this before submitting a PR?